src/test/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineServicesTest.java

```diff
@@ -61,7 +61,7 @@ public void setTrafficTest() throws ProcessRunnerException {
     appEngineService.setTraffic(configuration);
 
     List<String> args =
-        Arrays.asList("services", "set-traffic", "myService", "--splits", "v1=0.3,v2=0.7",
+        Arrays.asList("services", "set-traffic", "myService", "--splits", "v1=0.3", "v2=0.7",
```

Comment:
ohh. good eye! I think my change may have subtly broke this. It accepts the following format:

--splits v2=.5,v1=.5

With my update it will be space separated. I'll revisit this.